packages/components/src/Drawer/Drawer.css

```diff
@@ -18,6 +18,10 @@
   @media (--medium-screens), print {
     position: relative;
     z-index: auto;
+    width: 280px;
```

Comment:
Can we move these to css custom properties and use a calc based off of the base size?

Example:

```
:root {
   --drawer-width: calc(var(--space-base) * 17.5);
}
```

then in `@media (--wide-screens)` redeclare it as `--drawer-width: calc(var(--space-base) * 26.25);`

Reply:
ooooo can do!